client/src/components/CardContainer/CardContainer.module.scss

```diff
@@ -0,0 +1,100 @@
+.cardContainer {
```

Comment:
Transfer all styled needed (except colors and font-sizes) to your js file and delete SCSS file.